sdk/core/Azure.Core/src/RequestFailedException.cs

```diff
@@ -218,18 +224,9 @@ internal static bool TryExtractErrorContent(Response response, out ResponseError
             data = null;
             try
             {
-                string? content = null;
-                if (response.ContentStream != null && response.ContentStream.CanSeek)
-                {
-                    content = response.Content.ToString();
-                }
-                else
-                {
-                    // this path should only happen in exceptional cases such as when
-                    // the RFE ctor was called directly by client or customer code with an un-buffered response.
-                    // Generated code would never do this.
-                    content = ReadContent(response);
```

Comment:
Is the implication of removing this that we can never reach this code path anymore, even in the case described?

Reply:
Yes, this shouldn't be possible anymore apart from the user doing something weird with the stream.